api/app/sfms/fwi_processor.py

```diff
@@ -15,6 +15,10 @@ def calculate_dc(dc_ds: WPSDataset, temp_ds: WPSDataset, rh_ds: WPSDataset, prec
     rh_array, _ = rh_ds.replace_nodata_with(0)
     precip_array, _ = precip_ds.replace_nodata_with(0)
 
+    # Due to warping of the rh dataset, rh values can exceed 100 which breaks the calculation.
```

Comment:
I wonder if we should do this correction in `warp_to_match` to keep it in one place. Perhaps it's an optional parameter only applied to rh.